Assets/Scripts/IntroCutsceneController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UnityEngine;
+
+public class IntroCutsceneController : MonoBehaviour
+{
+    public AudioSource labChairSfx;
+    public AudioSource bemisVoice;
+    public AudioClip bemisDialogue2;
+    public AudioClip enterSimulationSfx;
+    public Animator simulationBoothLightingAnim;
+
+    public float spaceStationFlightSpeed = 10f;
+    
+    private static readonly int EnterSimulationBoothTrigger = Animator.StringToHash("EnterSimulationBooth");
+    private static readonly int StartSimulationTrigger = Animator.StringToHash("StartSimulation");
+    private static readonly int Rotation = Shader.PropertyToID("_Rotation");
+
+    private bool skyboxShouldRotate = true;
+
+    private void Awake()
+    {
+        Cursor.visible = false;
+        Cursor.lockState = CursorLockMode.Locked;
+    }
+
+    private void Update()
+    {
+        // press escape to skip cutscene
+        if (Input.GetKeyDown(KeyCode.Escape)) LoadFirstLevel();
+        
+        // rotate the skybox to simulate space station moving through space when player looks out the window
+        if (skyboxShouldRotate) RenderSettings.skybox.SetFloat(Rotation, Time.time * spaceStationFlightSpeed);
+    }
+
+    public void StartBemisDialogue()
+    {
+        bemisVoice.Play();
+        Invoke(nameof(EnterSimulationBooth), bemisVoice.clip.length);
```

Comment:
Rewrite this whole thing as a Coroutine (you can wait for a certain time period or based on a predicate -> ie until bemis audio source isn't playing anymore).

Reply:
Punting this to post Gold Master: https://chilipot.atlassian.net/browse/PROP-139